library/std/src/fs.rs

```diff
@@ -116,6 +116,19 @@ pub struct File {
     inner: fs_imp::File,
 }
 
+/// An enumeration of possible errors which can occur while trying to acquire a lock
+/// from the [`try_lock`] method and [`try_lock_shared`] method on a [`File`].
+///
+/// [`try_lock`]: File::try_lock
+/// [`try_lock_shared`]: File::try_lock_shared
+#[unstable(feature = "file_lock", issue = "130994")]
+pub enum TryLockError {
```

Comment:
Could the error type also implement core::error::Error?

Reply:
yep! I just added that

Reply:
I can't find it in this file.

Reply:
Oh sorry, I reverted it due to the discussion in the tracking issue about omitting the From implementation and just sticking to the signature change. Want me to add the Error impl back?